pydrive2/drive.py

```diff
@@ -46,4 +46,4 @@ def GetAbout(self):
 
         :returns: A dictionary of Google Drive information like user, usage, quota etc.
         """
-        return self.auth.service.about().get().execute(http=self.http)
+        return self.auth.service.about().get().execute(http=self.http)
```

Comment:
please, let's not change this - it unrelated to the PR (and actually it's better to have a newline at the EOF)